Felienne/hedy
# Add option to show/hide the fields that are already translated #531

**Description**

Add option to show only the fields that need translation or show all fields. Note that the fields that need translation are fields that 1) have empty translation or 2) have translation equivalent to the original. The second condition covers for cases in which the English properties are copied to the yaml file of another language.

**Fix for**

#531 

**How to test**

If you test this locally, please make sure to run the generate-typescript.
- Run Hedy and open the translation tool: 0.0.0.0:5000/translate/en/bg and check the button behaviour.
- Try translating a field and download the files (with both translated fields hidden and shown). Check that the files contain ALL fields.

**Checklist**

If you're unsure about any of these, don't hesitate to ask. We're here to help!
